### PR TITLE
app-misc/clipboard: new package, add 0.8.1

### PR DESCRIPTION
**Pull request by quincyf467**
I think this package is useful.  If the current directory "app-misc" is not suitable I'll change it to what is recommended.

### CODE REVIEW

**Comment by quincyf467** [comment]
> ## Pull request CI report
> 
> _Report generated at_: 2023-04-20 02:38 UTC _Newest commit scanned_: [1c5ce15](https://github.com/gentoo/gentoo/commit/1c5ce15be628ccba5c415a34094fb37757d5eab4) _Status_: x **broken**
> 
> New issues caused by PR: https://qa-reports.gentoo.org/output/gentoo-ci/408714e220/output.html#app-misc/Clipboard
> 
> There are existing issues already. Please look into the report to make sure none of them affect the packages in question: https://qa-reports.gentoo.org/output/gentoo-ci/408714e220/output.html

?   The USE flag "lto" is being provided by 27 packages in the gentoo tree right now.  I'm confused

**Comment by ceamac** [comment]
Please check [Local and Global USE Flags](https://devmanual.gentoo.org/general-concepts/use-flags/index.html#local-and-global-use-flags) and the following _USE Flag Descriptions_

**Comment by quincyf467** [comment]
@ceamac Thanks for the tip!

**Comment by Slackadays** [comment]
@quincyf467 You could change the description to `Cut, copy, and paste anything in your terminal` as that's what we've used elsewhere in other packages.

File: app-misc/Clipboard/files/Clipboard-0.6.0-libdestination-and-disable-lto-and-git.patch [hunk 1]
```diff
@@ -54,9 +54,25 @@ diff -ur '--color=auto' Clipboard-0.6.0/src/clipboard/CMakeLists.txt Clipboard-0
...
+     PROPERTY BUILD_RPATH
+-    "${CMAKE_INSTALL_PREFIX}/lib"
++    "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}"
+   )
+   set_property(
+     TARGET clipboard
+     APPEND
+     PROPERTY INSTALL_RPATH
+-    "${CMAKE_INSTALL_PREFIX}/lib"
++    "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}"
```
Comment (a17r) [inline app-misc/Clipboard/files/Clipboard-0.6.0-libdestination-and-disable-lto-and-git.patch]: Bad idea, `CMAKE_INSTALL_LIBDIR` may be absolute. Use `CMAKE_INSTALL_FULL_LIBDIR` exclusively instead.

**Pull request closed without merging**